ess/src/API/EMBC.ESS/Managers/Submissions/SubmissionsManager.cs

```diff
@@ -225,15 +225,19 @@ public async Task<VerifySecurityQuestionsResponse> Handle(VerifySecurityQuestion
 
         public async Task<VerifySecurityPhraseResponse> Handle(VerifySecurityPhraseQuery query)
         {
-            //bool maskSecurityPhrase = false;
-            //var file = await evacuationRepository.Read(query.FileId, maskSecurityPhrase);
-            //VerifySecurityPhraseResponse ret = new VerifySecurityPhraseResponse
-            //{
-            //    IsCorrect = file.SecurityPhrase.ToLower().Equals(query.SecurityPhrase.ToLower())
-            //};
-            //return ret;
-            await Task.CompletedTask;
-            throw new NotImplementedException("Need to fix this handler to call CASE repository, not evacuation repository");
+            var file = (await caseRepository.QueryCase(new EvacuationFilesQuery
+            {
+                FileId = query.FileId,
+                MaskSecurityPhrase = false
+            })).Items.Cast<Resources.Cases.EvacuationFile>().FirstOrDefault();
+
+            if (file == null) throw new Exception($"Evacuation File {query.FileId} not found");
+
+            VerifySecurityPhraseResponse ret = new VerifySecurityPhraseResponse
+            {
+                IsCorrect = file.SecurityPhrase.ToLower().Equals(query.SecurityPhrase.ToLower())
```

Comment:
I still recommend to use the Equals overload for string comparison here and security questions verifications